bodhi/server/templates/update.html

```diff
@@ -685,19 +775,14 @@ <h5 class="m-t-2">
                 </div>
               </div>
 
-              % if request.registry.settings.get('ci.required'):
+              % if request.registry.settings.get('greenwave_enabled'):
               <div class="p-b-1">
                 <div>
-                  % if request.registry.settings.get('ci.url'):
-                    <a href="${request.registry.settings.get('ci.url')}" title="More information about CI">
-                      <strong>CI Status</strong>
-                    </a>
-                  % else:
-                    <strong>CI Status</strong>
-                  % endif
+                  <strong>Gating Status</strong>
                 </div>
                 <div>
-                  ${self.util.ci_status2html(update.ci_status.description) | n}
+                  ${self.util.gating_status2html(update.greenwave_policies_satisfied,
+                    update.greenwave_summary_string) | n}
```

Comment:
Again, I think the existing field is best to keep here too as it allows more states than a bool.